Sources/SwifterSwift/SwiftStdlib/SequenceExtensions.swift

```diff
@@ -316,15 +314,15 @@ public extension Sequence where Element: Hashable {
     }
 }
 
-// MARK: - Methods (Numeric)
+// MARK: - Methods (AdditiveArithmetic)
 
-public extension Sequence where Element: Numeric {
+public extension Sequence where Element: AdditiveArithmetic {
     /// SwifterSwift: Sum of all elements in array.
     ///
     ///        [1, 2, 3, 4, 5].sum() -> 15
     ///
     /// - Returns: sum of the array's elements.
     func sum() -> Element {
-        return reduce(into: 0, +=)
+        return reduce(.zero, +)
```

Comment:
Why this? To me, this seems like a regression in performance, minor but still ...

Reply:
`0` is an `Int`, `.zero` is the actual element type.
`reduce(into:_:)` passes an `inout` variable, which means that there's added indirection via a reference. Since it's (probably) just a simple value like an `Int` it should be faster to copy the value instead of needing to dereference it.

Reply:
Yeah, it shouldn't be a huge problem in this case I think